src/index.js

```diff
@@ -80,6 +81,7 @@ ReactCodepen.defaultProps = {
   defaultTab: 'css,result',
   height: 300,
   preview: true,
+  editable: true,
```

Comment:
I feel this could be opt-in and defaulted to false?

Reply:
changed